docs/guides/result_storage.mdx

```diff
@@ -5,106 +5,96 @@ title: Result Storage
 
 ## Key-value store
 
-The key-value store is used for saving and reading data records or files. Each data record is represented by a unique key and associated with a MIME
-content type. Key-value stores are ideal for saving screenshots of web pages, PDFs or to persist the state of crawlers.
+The key-value store is used for saving and reading data records or files. Each data record is represented by a unique key and associated with a MIME content type. Key-value stores are ideal for saving screenshots of web pages, PDFs or to persist the state of crawlers.
 
-Each actor run is associated with a **default key-value store**, which is created exclusively for the actor run. By convention, the actor run input
-and output is stored in the default key-value store under the `INPUT` and `OUTPUT` key, respectively. Typically the input and output is a JSON file,
-although it can be any other format.
+Each Crawlee project is associated with a **default key-value store**. By convention, the project input and output are stored in the default key-value store under the `INPUT` and `OUTPUT` keys respectively. Typically, both input and output are JSON files, although they could be any other format.
 
-In Crawlee, the key-value store is represented by the <ApiLink to="core/class/KeyValueStore">`KeyValueStore`</ApiLink> class. In order to simplify access to the default
-key-value store, Crawlee also provides <ApiLink to="apify/class/Actor#getValue">`Actor.getValue()`</ApiLink> and <ApiLink to="apify/class/Actor#setValue">`Actor.setValue()`</ApiLink> functions.
+In Crawlee, the key-value store is represented by the <ApiLink to="core/class/KeyValueStore">`KeyValueStore`</ApiLink> class. In order to simplify access to the default key-value store, Crawlee also provides <ApiLink to="core/class/KeyValueStore#getValue">`KeyValueStore.getValue()`</ApiLink> and <ApiLink to="core/class/KeyValueStore#setValue">`KeyValueStore.setValue()`</ApiLink> functions.
 
-In local configuration, the data is stored in the directory specified by the `APIFY_LOCAL_STORAGE_DIR` environment variable as follows:
+The data is stored in the directory specified by the `CRAWLEE_STORAGE_DIR` environment variable as follows:
 
 ```
-{APIFY_LOCAL_STORAGE_DIR}/key_value_stores/{STORE_ID}/{KEY}.{EXT}
+{CRAWLEE_STORAGE_DIR}/key_value_stores/{STORE_ID}/{KEY}.{EXT}
 ```
 
-Note that `{STORE_ID}` is the name or ID of the key-value store. The default key value store has ID `default`, unless you override it by setting the
-`APIFY_DEFAULT_KEY_VALUE_STORE_ID` environment variable. The `{KEY}` is the key of the record and `{EXT}` corresponds to the MIME content type of the
-data value.
+:::note
+
+`{STORE_ID}` is the name or the ID of the key-value store. The default key-value store has ID `default`, unless you override it by setting the `APIFY_DEFAULT_KEY_VALUE_STORE_ID` environment variable. The `{KEY}` is the key of the record and `{EXT}` corresponds to the MIME content type of the data value.
```

Comment:
This remark about Apify might be better in the platform section?

Reply:
Well - we discussed that crawlee guides should not mention Apify in any way, that there is platform guide, which would be more verbose, and all Apify-related stuff would be there. In this case - there should probably be some other ENV var that could be used with Crawlee locally to override the default storage name. So it's not missed - it should be replaced...

Reply:
yop, we will need most of the env vars to work with `CRAWLEE_` prefix, no need to care about it now as they dont exist yet, but thats something to fix in following week or two